kubernetes/linux/setup.sh

```diff
@@ -67,7 +67,7 @@ sudo tdnf install jq-1.6-1.cm2 -y
 #used to setcaps for ruby process to read /proc/env
 sudo tdnf install libcap -y
 
-sudo tdnf install telegraf-1.26.0 -y
+sudo tdnf install telegraf-1.27.2 -y
```

Comment:
@jatakiajanvi12 , 1. did you verify functionality (disk, diskio, network and prom metrics) works and there are no regressions? 2. Does this upgrade addresses all the critical and high telegraf vulns which we were added trivyignore? if yes, please remove the vulns which addressed from trivyignore so that we know the remaining ones to address.

Reply:
1. Yes all of them are present. 
2. Updated the PR